Python/frozen_hello.h

```diff
@@ -1,13 +1,12 @@
-/* Generated with Tools/freeze/regen_frozen.py */
-static unsigned char M___hello__[] = {
-    227,0,0,0,0,0,0,0,0,0,0,0,0,
-    0,0,0,0,2,0,0,0,64,0,0,0,115,
-    16,0,0,0,100,0,90,0,101,1,100,1,131,
-    1,1,0,100,2,83,0,41,3,84,122,12,72,
-    101,108,108,111,32,119,111,114,108,100,33,78,41,
-    2,90,11,105,110,105,116,105,97,108,105,122,101,
-    100,218,5,112,114,105,110,116,169,0,114,2,0,
-    0,0,114,2,0,0,0,218,4,110,111,110,101,
-    218,8,60,109,111,100,117,108,101,62,1,0,0,
-    0,115,2,0,0,0,4,1,
+/* Auto-generated by Programs/_freeze_importlib.c */
+const unsigned char _Py_M__hello[] = {
```

Comment:
Not sure making this non-static is good.  The _freeze_importlib could grow a command line option to make it static or non-static.